src/thread/vita/SDL_systhread.c

```diff
@@ -59,7 +59,7 @@ int SDL_SYS_CreateThread(SDL_Thread *thread, void *args)
         return SDL_SetError("sceKernelCreateThread() failed");
     }
 
-    sceKernelStartThread(thread->handle, 4, &args);
+    sceKernelStartThread(thread->handle, 4, &thread);
```

Comment:
This needed a change. SDL2 now passes the whole thread as argument, not only the arguments themselves.